.github/workflows/build.yml

```diff
@@ -755,3 +755,50 @@ jobs:
         if: contains(needs.*.result, 'failure')
         run: |
           echo "One of the dependent jobs have failed. You may need to re-run it." && exit 1
+
+  replay_metrics:
```

Comment:
Could we move this into a dedicated workflow, that only runs when a file in `packages/replay` has been changed in a PR?

Reply:
I've had it like that before but moved due to the need to build all packages. Or at least I must have missed something because just building tracing & metrics package didn't produce the correct result. You can see the previous version in this commit: [`0af7c22` (#6611)](https://github.com/getsentry/sentry-javascript/pull/6611/commits/0af7c22e023ded78e01b94558db378d2d933a41f) - see file metrics.yml

Maybe you have an idea what needs to be changed there so that [`yarn build:bundle`](https://github.com/getsentry/sentry-javascript/pull/6611/files#diff-26636b8b6ab6555a72ec9e957d1a7c570a75ebb64a4b5f949ec2ea37d926ab4bR10) on `metrics` & `tracing` would produce a correct *.min.js without running a full build?

Reply:
We just discussed this again and it's fine to leave it in this workflow. But can we make the execution of this job depend on a label we manually add to the PR where we want to run these measurements? So that it only runs if we add the `CI-Overhead-Measurements` label

Reply:
That could be done by adding a condition on the job: 
```yaml
if: contains(github.event.pull_request.labels.*.name, 'CI-Overhead-Measurements')
```

However, in that case, it would probably be useful to get a PR-labeled trigger which in turn would mean the whole build job would run again after any label is added to a PR. Not sure that's desirable, though.
```yaml
on:
  pull_request:
    types: [ labeled ]
```

Reply:
> That could be done by adding a condition on the job: 
> ```yaml
> if: contains(github.event.pull_request.labels.*.name, 'CI-Overhead-Measurements')
> ```
> 
> However, in that case, it would probably be useful to get a PR-labeled trigger which in turn would mean the whole build job would run again after any label is added to a PR. Not sure that's desirable, though.
> ```yaml
> on:
>   pull_request:
>     types: [ labeled ]
> ```

I think it's ok to start to just have the if condition. You need to either add the label at pr creation, or make sure to manually rerun after adding the label - good enough for a start!

Reply:
ok, added the condition